lib/binance/websocket/ws_client.ex

```diff
@@ -91,8 +91,8 @@ defmodule Binance.WebSocket.WSClient do
         {:ok, state}
       end
 
-      def handle_info(:keep_alive, state) do
-        {:ok, _} = Binance.keep_alive_listen_key()
+      def handle_info(:keep_alive, %{listen_key: listen_key} = state) do
+        {:ok, _} = Binance.keep_alive_listen_key(listen_key)
```

Comment:
Can you explain this part?

Reply:
`Binance.keep_alive_listen_key` for Spot expects an argument but I forgot to update that.